README.md

```diff
@@ -1 +1,76 @@
-# angular-controlleras-readme
\ No newline at end of file
+# Using controllerAs
+
+## Overview
+
+We've learned already what `$scope` is and the magic it provides with our view. However, before we start going `$scope` mad, it's important we learn about the best practices when it comes to controllers and our views.
```

Comment:
really great introduction - this draws a perfect line from what we've already learn to what our objectives are for this README